afs/media/js/utils/report.js

```diff
@@ -40,12 +40,13 @@ define([
             return $.ajax({
                 type: "DELETE",
                 url: arches.urls.tile,
-                data: JSON.stringify(tile.getData())
```

Comment:
What was the rationale behind this change?

Reply:
Here is the [SO post](https://stackoverflow.com/questions/39042085/uncaught-typeerror-ajax-success-is-not-a-function) that led me to make this change. It looks like we're using jQuery 3.6.1 and that the method form of success has been deprecated since 1.8.